$OpenBSD: patch-configure,v 1.2 2015/08/24 23:02:31 bcallah Exp $

configure is stupid. Force it to do the right thing.

--- configure.orig	Mon Aug 24 16:56:13 2015
+++ configure	Mon Aug 24 18:24:30 2015
@@ -3572,7 +3572,7 @@ fi
 
 
 #   fftw3f
-deps_packages+=" fftw3f"
+#deps_packages+=" fftw3f"
 
 #   ALSA
 
@@ -3597,20 +3597,8 @@ _ACEOF
 
 #   pulseaudio
 
-# Check whether --with-pulseaudio was given.
-if test "${with_pulseaudio+set}" = set; then :
-  withval=$with_pulseaudio;
-fi
+use_pulseaudio=1
 
-if test "x$with_pulseaudio" == "xno"; then :
-  # then
-	use_pulseaudio=0
-else
-  # else
-	use_pulseaudio=1
-	deps_packages+=" libpulse-simple"
-fi
-
 cat >>confdefs.h <<_ACEOF
 #define USE_PULSEAUDIO $use_pulseaudio
 _ACEOF
@@ -3618,20 +3606,8 @@ _ACEOF
 
 #   sndfile
 
-# Check whether --with-sndfile was given.
-if test "${with_sndfile+set}" = set; then :
-  withval=$with_sndfile;
-fi
+use_sndfile=1
 
-if test "x$with_sndfile" == "xno"; then :
-  # then
-	use_sndfile=0
-else
-  # else
-	use_sndfile=1
-	deps_packages+=" sndfile"
-fi
-
 cat >>confdefs.h <<_ACEOF
 #define USE_SNDFILE $use_sndfile
 _ACEOF
@@ -3814,7 +3790,7 @@ if test -n "$DEPS_CFLAGS"; then
   pkg_cv_DEPS_CFLAGS=`$PKG_CONFIG --cflags " $deps_packages " 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
-  pkg_failed=yes
+  pkg_failed=no
 fi
  else
     pkg_failed=untried
@@ -3831,7 +3807,7 @@ if test -n "$DEPS_LIBS"; then
   pkg_cv_DEPS_LIBS=`$PKG_CONFIG --libs " $deps_packages " 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
-  pkg_failed=yes
+  pkg_failed=no
 fi
  else
     pkg_failed=untried
